### DISCUSSION TITLE
寄了，不知道为什么QAQ

### DISCUSSION THREAD

**Discussion by Nuru-Banmian** [LmeSzinc/AzurLaneAutoScript#2700] [Q&A] [answered]
本来不想麻烦大佬的，但自己试了好几种办法了还是解决不了，也翻过前面类似的求助贴了，还是解决不了

![JCE8}OK}3}`9 BP T~((PC8](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/127472692/adb6245e-d898-44d2-8de8-30c429734e1c)

**Comment by weduolijia** [comment, marked as answer]
Repository源挂掉了，需要重新更新deploy.yaml内的源

**Reply by Nuru-Banmian** [reply]
方便留一下操作方法吗

**Reply by Nuru-Banmian** [reply]
bbzl，我把deploy.yaml文件直接删了再启动alas就好了，虽然全都变英文了

**Reply by Nuru-Banmian** [reply]
大佬能留一下操作方法嘛

**Reply by ZLCNBA** [reply]
> 大佬能留下来操作方法嘛

我删掉了deploy.yaml里Repository:后面的地址，即是中文也可以打开，但我不知道会发生什么隐藏的问题，因为我什么甚至不知道Repository源是用来的干嘛的

**Reply by Nuru-Banmian** [reply]
> > 大佬能留下来操作方法嘛
> 
> 我删掉了deploy.yaml里Repository:后面的地址，即是中文也可以打开，但我不知道会发生什么隐藏的问题，因为我什么甚至不知道Repository源是用来的干嘛的

好的我试一下

**Reply by Nuru-Banmian** [reply]
没什么用欸
